### PR TITLE
fix side nav, undoing inadvertant changes

### PR DESCRIPTION
**Pull request by MichaelTrestman**

### CODE REVIEW

**Review by stbof — CHANGES_REQUESTED** [review]
It looks right. I'll approve after you merge master to pick up the 22.2 docs and propagate this fix to that version.

**Comment by MichaelTrestman** [comment]
@stbof looks ok now, eh?

**Comment by stbof** [comment]
> @stbof looks ok now, eh?

Yes. In the future, to request a re-review could you click this double circular arrow? 

Screen Shot 2022-08-17 at 11 28 50 AM

**Pull request merged**
